flask-backend/api/routes/commonwords.py

```diff
@@ -0,0 +1,412 @@
+'''
```

Comment:
Logic is appropriate.
Nit: format the code properly, remove redundant line spaces.

Reply:
Done @shivanshu1333  sir!